### PR TITLE
SRVCOM-1661: release notes and build error fix

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVCOM-1661 (add release notes)
Also fixes issue reported by @bergerhoffer 

Applies for OCP 4.6+ (requires manual CP for versions older than 4.9)

Direct preview links:
OCP
- https://deploy-preview-43021--osdocs.netlify.app/openshift-enterprise/latest/serverless/install/install-serverless-operator.html
- https://deploy-preview-43021--osdocs.netlify.app/openshift-enterprise/latest/serverless/serverless-release-notes.html#serverless-rn-1-21-0_serverless-release-notes
- https://deploy-preview-43021--osdocs.netlify.app/openshift-enterprise/latest/serverless/serverless-release-notes.html#serverless-rn-1-20-0_serverless-release-notes (updated to add FIPS note)
- https://deploy-preview-43021--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-kafka-developer.html#serverless-kafka-developer-broker (updated to add FIPS note)

OSD
- https://deploy-preview-43021--osdocs.netlify.app/openshift-dedicated/latest/serverless/install/install-serverless-operator.html
- https://deploy-preview-43021--osdocs.netlify.app/openshift-dedicated/latest/serverless/serverless-release-notes.html#serverless-rn-1-21-0_serverless-release-notes
- https://deploy-preview-43021--osdocs.netlify.app/openshift-dedicated/latest/serverless/serverless-release-notes.html#serverless-rn-1-20-0_serverless-release-notes (updated to add FIPS note)
- https://deploy-preview-43021--osdocs.netlify.app/openshift-dedicated/latest/serverless/develop/serverless-kafka-developer.html#serverless-kafka-developer-broker (updated to add FIPS note)

### CODE REVIEW

File: modules/serverless-rn-1-21-0.adoc [hunk 1]
```diff
@@ -11,18 +11,20 @@
...
-* {ServerlessProductName} now uses Kourier 0.27.
-* {ServerlessProductName} now uses Knative `kn` CLI 0.27.
-* {ServerlessProductName} now uses Knative Kafka 0.27.
+* {ServerlessProductName} now uses Knative Serving 1.0
+* {ServerlessProductName} now uses Knative Eventing 1.0.
+* {ServerlessProductName} now uses Kourier 1.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 1.0.
+* {ServerlessProductName} now uses Knative Kafka 1.0.
 * The `kn func` CLI plug-in now uses `func` 0.21.
+* The Kafka sink is now available as a Technology Preview.
```
Comment (mgencur) [inline modules/serverless-rn-1-21-0.adoc:20]: The versions look correct.

**Comment by mgencur** [comment]
Looks good to me now.
QE Approved.

File: modules/serverless-rn-1-21-0.adoc [hunk 2]
```diff
@@ -11,19 +11,49 @@
...
-// check versions
+* The Kafka sink is now available as a Technology Preview.
+
 * The Knative open source project has begun to deprecate camel-cased configuration keys in favor of using kebab-cased keys consistently. As a result, the `defaultExternalScheme` key, previously mentioned in the {ServerlessProductName} 1.18.0 release notes, has been deprecated and replaced by the `default-external-scheme` key. Usage instructions for the key remain the same.
 
 [id="fixed-issues-1-21-0_{context}"]
 == Fixed issues
-// add a version, e.g. 1-20-0
+
+* In {ServerlessProductName} 1.20.0, there was an event delivery issue affecting the use of `kn event send` to send events to a service. This issue has been fixed.
```
Comment (ahardin-rh) [inline modules/serverless-rn-1-21-0.adoc:27]: s/has been/is now

File: modules/serverless-rn-1-20-0.adoc [hunk 3]
```diff
@@ -19,6 +19,12 @@
 * The `kn func` CLI plug-in now uses `func` 0.20.
 
 * The Kafka broker is now available as a Technology Preview.
++
+[IMPORTANT]
+====
+The Kafka broker Technology Preview is not supported on FIPS.
```
Comment (ahardin-rh) [inline modules/serverless-rn-1-20-0.adoc:25]: for translation purposes, we should call it a Technology Preview feature and not just a Technology Preview. Maybe reword this to:

The Kafka Broker, a Technology Preview feature, is currently not supported on FIPS.
or
The Kafka Broker, which is currently in Technology Preview, is not supported on FIPS.

File: serverless/develop/serverless-kafka-developer.adoc [hunk 4]
```diff
@@ -43,10 +46,15 @@ include::modules/serverless-kafka-source-yaml.adoc[leveloffset=+2]
...
 
+If a cluster administrator has configured your {ServerlessProductName} deployment to use Kafka broker as the default broker type, xref:../../serverless/develop/serverless-using-brokers.adoc#serverless-using-brokers-creating-brokers[creating a broker by using the default settings] creates a Kafka-based `Broker` object. If your {ServerlessProductName} deployment is not configured to use Kafka broker as the default broker type, you can still use the following procedure to create a Kafka-based broker.
+
 :FeatureName: Kafka broker
 include::snippets/technology-preview.adoc[leveloffset=+2]
 
-If a cluster administrator has configured your {ServerlessProductName} deployment to use Kafka broker as the default broker type, xref:../../serverless/develop/serverless-using-brokers.adoc#serverless-using-brokers-creating-brokers[creating a broker by using the default settings] creates a Kafka-based `Broker` object. If your {ServerlessProductName} deployment is not configured to use Kafka broker as the default broker type, you can still use the following procedure to create a Kafka-based broker.
+[IMPORTANT]
+====
+The Kafka broker Technology Preview is not supported on FIPS.
```
Comment (ahardin-rh) [inline serverless/develop/serverless-kafka-developer.adoc:56]: Same here:
The Kafka broker, a Technology Preview feature, is currently not supported on FIPS.
or
The Kafka broker, which is currently in Technology Preview, is not supported on FIPS.

**Comment by ahardin-rh** [comment]
@abrennan89 A few suggestions from me. Otherwise, looks great!

File: modules/serverless-rn-1-21-0.adoc (hunk 2 above)
Comment (abrennan89) [reply modules/serverless-rn-1-21-0.adoc:27]: replaced in all instances, thanks!
